tests/test_dataset.py

```diff
@@ -0,0 +1,253 @@
+import os
+import posixpath
+
+import pytest
+
+import audeer
+import audiofile
+import audb
+import audformat
+import audbcards
+import audplot
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+BUILD = audeer.path('..', 'build', 'html')
+
+
+def test_dataset(cache, tmpdir, db):
+    dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION)
+
+    # __init__
+    assert dataset.name == pytest.NAME
+    assert dataset.version == pytest.VERSION
+    assert dataset.repository == pytest.REPOSITORY
+    expected_header = audb.info.header(
+        pytest.NAME,
+        version=pytest.VERSION,
+        cache_root=cache,
+    )
+    assert str(dataset.header) == str(expected_header)
+    expected_deps = audb.dependencies(
+        pytest.NAME,
+        version=pytest.VERSION,
+        cache_root=cache,
+    )
+    pd.testing.assert_frame_equal(dataset.deps(), expected_deps())
+
+    # archives
+    assert dataset.archives == str(len(db.files))
```

Comment:
The number of archives per database does not need to reflect the number of media files.
We can get the correct number by checking the number of archives listed in the dependency table for media files, e.g. something like:
```suggestion
    expected_df = expected_deps()
    pd.testing.assert_frame_equal(dataset.deps(), expected_df)

    # archives
    expected_archives = str(
        len(expected_df.loc[expected_deps.media].archive.unique())
    )
    assert dataset.archives == expected_archives
```